pisek/__main__.py

```diff
@@ -243,6 +243,13 @@ def add_argument_dataset(parser):
     config_subparsers.add_parser(
         "update", help="update config to newest version (replaces the config)"
     )
+    parser_export = config_subparsers.add_parser(
+        "export", help="creates an organization independent config"
+    )
+    parser_export.add_argument(
+        "filename",
+        help="Filename of the exported config"
```

Comment:
Most other help messages start with a lowercase character.
This message would be more similar to the help message of `--config-filename`:

```suggestion
        help="name of the exported configuration file"
```